api/helpers/get-queue-object.js

```diff
@@ -31,3 +26,31 @@ module.exports = {
     return exits.success(queue);
   }
 };
+
+// I don't like this approach
+// we should use a real Bull queue with in memory Redis or something
+// Blocked by https://github.com/OptimalBits/bull/issues/962 I guess
+const mockQueue = {
```

Comment:
I don't like this because the tests I wrote kind of rely on Bull doing it's job properly. By creating this mock queue, the tests are not actually testing the correct thing.

Perhaps I wrote my tests wrong? Idk, WDYT @halkeye ?

Reply:
There's a couple mock redis libraries that can override create server. I can certainly look into them. I don't like mocking when I can get away with just swapping out a single function for exactly that reason

Reply:
Yeah I had a look at those aswell. In the issue linked it said the mock redis library were not feature complete and lacked some stuff Bull required. Those are from a few years ago though so might have changed